pkg/i2gw/providers/istio/resource_reader.go

```diff
@@ -58,6 +60,25 @@ func (r *reader) readResourcesFromCluster(ctx context.Context) (*storage, error)
 	return &res, nil
 }
 
+func (r *reader) readResourcesFromFile(_ context.Context, filename string) (*storage, error) {
```

Comment:
I'd suggest to remove `_ context.Context` from params, it's not used and not needed (this method doesn't satisfy any interface that requires such param).